lib/queries.js

```diff
@@ -125,21 +171,27 @@ export async function createMandate(classification, governingBodiesInTime) {
     ${PREFIXES}
 
     INSERT DATA {
-        ${linksToGoverningBodies}
+      ${linksToGoverningBodies}
 
-        ${sparqlEscapeUri(uri)} a mandaat:Mandaat ;
-          mu:uuid ${sparqlEscapeString(id)} ;
-          org:role ${sparqlEscapeUri(classification)} .
+      ${sparqlEscapeUri(uri)} a mandaat:Mandaat ;
+        mu:uuid ${sparqlEscapeString(id)} ;
+        org:role ${sparqlEscapeUri(role)} .
     }
   `);
 
   return uri;
 }
 
-export async function createBestuursfunctie(
-  classification,
-  governingBodyInTime
-) {
+/**
+ * Create a new bestuursfunctie, with a given role, for the provided
+ * time-specialised governing body.
+ * @param {string} role - the URI of the role for the new bestuursfunctie.
+ * @param {string} governingBodyInTime ${2} - the URI of the governing body of
```

Comment:
**Question**: What is the purpose of `${2}`?

Reply:
No idea, it is something automatically added by my completions system, and  I have not gotten around to look further into it. I just forget to remove this one.